src/App.js

```diff
@@ -1,61 +1,59 @@
 import './styles/reset.css'
 import './styles/index.css'
 
-import initialStoreItems from './store-items'
-
-/*
-Here's what a store item should look like
-{
-  id: '001-beetroot',
-  name: 'beetroot',
-  price: 0.35
-}
-
-What should a cart item look like? 🤔
-*/
+import {useState} from 'react'
+import Header from './Header'
+import Footer from './Footer'
+import Cart from './Cart'
+import Detail from './Detail'
 
-console.log(initialStoreItems)
 
 export default function App() {
-  // Setup state here...
+  
+const [cartItems, setCartItems] = useState([])
+const [showingDetail, setShowingDetail] = useState(false)
+const [detailItem, setDetailItem] = useState({})
+
+const alreadyInCart = (item) => {
+  return cartItems.find(cartItem => cartItem.name === item.name) !== undefined
+
+}
+
+const addToCart = (item) => {
+    if(alreadyInCart(item)) {
+  const updatedCart = cartItems.map(cartItem => cartItem.name === item.name ? {...cartItem, quantity: cartItem.quantity+= 1} : cartItem)
```

Comment:
I like the use of the ternary operator. I need to understand it better so I can start using it but I am still debating how it looks from a readability point of view.